package.json

```
@@ -1,12 +1,12 @@
 {
   "name": "Pacman",
   "version": "0.1.0",
-  "private": true,
   "license": "MIT",
   "author": "Batch 5 Pesto",
   "scripts": {
     "lint": "eslint .",
-    "lintfix": "eslint . --fix"
+    "lintfix": "eslint . --fix",
+    "prelint": "yarn && cd client && yarn && cd ../server && yarn && cd .. && ls"
```

Instead of this try this following
```suggestion
    "postinstall": "yarn install:client && yarn install:server",
    "install:client": "cd client && yarn",
    "install:server": "cd server && yarn",
```
^ By this you can run the installations separately or all at once

---

Thanks.